client/state/selectors/get-blocked-sites.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { map, pickBy } from 'lodash';
+
+/**
+ * Returns a list of site IDs blocked by the user
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Array}        Blocked site IDs
+ */
+export default function getBlockedSites( state ) {
+	return map( Object.keys( pickBy( state.reader.siteBlocks.items ) ), Number );
```

Comment:
might it be better to leave is as a map?
It looks like we only use it for the purposes of existence lookups.
I could even imagine returns `state.reader.siteBlocks.items`.

then the calling code could be:
```jsx
if ( blockedSites[ siteId ] ) {
   ...
}
```